src/main/resources/db/migration/V007__Envelope_upgrade.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TABLE process_events DROP COLUMN envelope_id;
+
+ALTER TABLE envelopes
+  ADD COLUMN container VARCHAR(50) NOT NULL DEFAULT 'sscs', -- absolutely not null. can drop default in next migration
```

Comment:
I would normally do this:
1. add nullable column
2. update existing rows
3. make column NOT NULL

Reply:
if you have something to update - perhaps. now it's enough to assign default. only one container is present too

Reply:
there should never be a container assigned by default if not provided by the app